src/lib/Scenes/Home/index.tsx

```diff
@@ -37,13 +38,27 @@ const ArtistsWorksForYouTab = 0
 const ForYouTab = 1
 const AuctionsTab = 2
 
+const screenSchemaForCurrentTab = currentSelectedTab => {
+  let screenType
+
+  if (currentSelectedTab === ArtistsWorksForYouTab) {
+    screenType = Schema.PageNames.HomeArtistsWorksForYou
+  } else if (currentSelectedTab === ForYouTab) {
+    screenType = Schema.PageNames.HomeForYou
+  } else if (currentSelectedTab === AuctionsTab) {
+    screenType = Schema.PageNames.HomeAuctions
+  }
+
+  return screenType
+}
+
 // This kills two birds with one stone:
 // It's necessary to wrap all tracks nested in this component, so they dispatch properly
 // Also, it'll only fire when the home screen is mounted, the only event we would otherwise miss with our own callbacks
-@screenTrack({
-  context_screen: Schema.PageNames.HomeArtistsWorksForYou,
+@screenTrack((props: Props) => ({
+  context_screen: screenSchemaForCurrentTab(props.initialTab),
```

Comment:
Wait, should this be `initialTab` or `selectedTab`?

Reply:
Great Q!

It should indeed `initialTab` because this event only gets fired once (and hence should reflect the first page it starts on). 

Any other changes are taken care of by the fireAnalytics method, also when changing tabs programmatically (and updating the `selectedTab` var).